back-end/config/LoaderFactory.js

```diff
@@ -0,0 +1,27 @@
+const AbstractLoader = require('./loader/AbstractLoader');
+
+class LoaderFactory {
+    constructor() {
+        this.loader = undefined;
+    }
+
+    getLoader() {
+        if (this.loader instanceof AbstractLoader) {
+            return this.loader;
+        }
+
+        switch (process.env.STORAGE) {
+            case 'firebase':
+                this.loader = require('./loader/Firebase');
```

Comment:
@RickvdStaaij I did this so only one or the other is loaded. If I were to load the Firebase config loader at the top of the document, the app would crash because it would fail validation (e.g. missing auth file). This way only the part that's needed is loaded. Works the same for the persister classes.

Reply:
Fine :+1: